modules/migration-mtc-release-notes-1-8-4.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-4_{context}"]
+= {mtc-full} 1.8.4 release notes
+
+[id=technical-changes-1-8-4_{context}]
+== Technical changes
+
+{mtc-full} ({mtc-short}) 1.8.4 has the following technical changes:
+
+* {mtc-short} 1.8.4 extends its dependency resolution to include support for using {oadp-first} 1.4.
+
+.Support for KubeVirt Virtual Machines with DirectVolumeMigration
```

Comment:
@rayfordj && @stillalearner: 

* Is there anything else you want included in technical changes?
* Is this enough for KubeVirt or is there more you want added?

Thanks

Reply:
@dymurray, anything to add here?

Reply:
Is this Tech or Dev preview? Once confirmed, i will add the required snippet
Thanks

Reply:
Dev